http_server/net_http/main.go

```diff
@@ -52,13 +54,26 @@ func squareHandler(w http.ResponseWriter, req *http.Request) {
 		fmt.Fprint(w, "num is not integer")
 		return
 	}
+	// 100以上の場合は、エラーとする
+	if num > 100 {
+		w.WriteHeader(http.StatusBadRequest)
+		fmt.Fprint(w, "num have to 100 or less")
+		return
+	}
+
 	// fmt.Sprintfでフォーマットに沿った文字列を生成できる。
 	fmt.Fprint(w, fmt.Sprintf("Square of %d is equal to %d", num, num*num))
 }
 
 // Bodyから数字を取得してその数字だけCounterをIncrementするハンドラー
 // DBがまだないので簡易的なもの
 func incrementHandler(w http.ResponseWriter, req *http.Request) {
+	if req.Method != http.MethodPost {
+		w.WriteHeader(http.StatusMethodNotAllowed)
+		fmt.Fprint(w, "This endpoint only accepts post requests. This is a " + req.Method + " request.")
```

Comment:
Goだと以下の様に、fmt.Fprintfという文字列のフォーマットが出来る関数があるので、そちらを使うのが良さそうです。
フォーマット関数の方が可読性が上がるかなーと個人的には思ってます。
```suggestion
		fmt.Fprintf(w, "This endpoint only accepts post requests. This is a %s request.", req.Method)
```